### DISCUSSION TITLE
Can no longer add item to new shopping lists

### DISCUSSION THREAD

**Discussion by RalfJung** [Abrynos/ShoppingList#105] [Q&A] [answered]
Since today I am having the weirdest bug... when I create a new shipping list and then try to add an item to it, nothing happens. I click "okay" to confirm the item, and then the item-add window closes but the item is not added to the shopping list. There's no error message, either.

I can still add items to old shopping lists. But not to new ones. That makes the app kind of useless...

I tried checking `adb logcat` to see if anything stands out, but not really. This is the log of when I click "okay":
```
03-28 18:58:08.514  3831  3859 D privacyfriendlyshoppinglist.secuso.org.privacyfriendlyshoppinglist.logic.shoppingList.persisteMETHOD=getById; RESULT=successful
03-28 18:58:08.520  3831  3859 D privacyfriendlyshoppinglist.secuso.org.privacyfriendlyshoppinglist.logic.product.persistence.iMETHOD=saveOrUpdate; RESULT=successful
03-28 18:58:08.520  1681  2003 W InputManager-JNI: Input channel object '730e836 privacyfriendlyshoppinglist.secuso.org.privacyfriendlyshoppinglist/privacyfriendlyshoppinglist.secuso.org.privacyfriendlyshoppinglist.ui.products.ProductsActivity (client)' was disposed without first being removed with the input manager!
03-28 18:58:08.538  3831  3860 D privacyfriendlyshoppinglist.secuso.org.privacyfriendlyshoppinglist.logic.product.persistence.iMETHOD=getAllEntities; RESULT=successful
03-28 18:58:08.543  3831  3861 D privacyfriendlyshoppinglist.secuso.org.privacyfriendlyshoppinglist.logic.shoppingList.persisteMETHOD=getById; RESULT=successful
```
The InputManager-JNI error also shows on successful adds to other lists so that can't be it.

When I "duplicate" a list, new new list also created empty. I assume that's not supposed to happen either?

Does anyone have any idea what might be causing this? It makes absolutely no sense to me...

**Comment by RalfJung** [comment, marked as answer]
Never mind, I confused multiple different apps that are all called "Shopping List" in F-Droid :joy:

**Reply by Abrynos** [reply]
Indeed. Reading your message was very confusing for a second 😂

**Reply by RalfJung** [reply]
Sorry for that. :)
